tools/jit/gen_unboxing.py

```diff
@@ -0,0 +1,169 @@
+# Generates CodegenUnboxingWrappers.cpp.
+# This generates static unboxing wrapper for ATen ops.
+import argparse
+import json
+import os
+from dataclasses import dataclass
+from tools.codegen.api import unboxing
+from tools.codegen.api.types import CppSignatureGroup
+from tools.codegen.context import method_with_native_function
+from tools.codegen.gen import parse_native_yaml
+from tools.codegen.model import NativeFunction, NativeFunctionsGroup
+from tools.codegen.utils import Target, FileManager, mapMaybe, make_file_manager
+from typing import Union, Sequence
+from typing_extensions import Literal
+
+
+# Generates CodegenFunctions.h & CodegenFunctions.cpp.
+@dataclass(frozen=True)
+class ComputeUnboxingFunctions:
+    target: Union[Literal[Target.DECLARATION], Literal[Target.DEFINITION]]
+
+    @method_with_native_function
+    def __call__(self, f: NativeFunction) -> str:
+        sig_group = CppSignatureGroup.from_native_function(
```

Comment:
is this used anywhere? It looks like it's just passed directly into `generate_unboxed_kernel_call` further down.

Reply:
Are you asking about `sig_group`? Yes we only get `sig` from it and pass to `generate_unboxed_kernel_call`.

Reply:
So maybe move it into the `else` block?

Reply:
Inlined `generate_unboxed_kernel_call` into `gen_unboxing.py`.